app/themes/base.js

```diff
@@ -1,24 +1,4 @@
-import {
-  green500,
-  green600,
-  green700,
-  purple500,
-  purple600,
-  purple700,
-  grey50,
-  grey100,
-  grey200,
-  grey300,
-  grey400,
-  grey500,
-  grey600,
-  grey700,
-  grey800,
-  grey900,
-  darkBlack,
-  fullBlack,
-  white
-} from 'material-ui/styles/colors'
+import {green, purple, grey, black, white} from '@material-ui/core/colors'
```

Comment:
this is unrelated / maybe related: apparently material-ui as a much improved theme structure, we might want to copy them more closely. @mischa-s was saying something about this?

Reply:
good thought - i'll break this out into another ticket i reckon and tackle it that way :)